pkg/proto/aksnodeconfig/v1/tlsbootstrappingconfig.proto

```diff
@@ -1,13 +1,26 @@
 syntax = "proto3";
 package aksnodeconfig.v1;
- 
+
+enum BootstrappingAuthMethod  {
+  //nolint:gosec // this is a const string to use in switch statements, not hardcoded credentials
+  UseTLSBootstrapToken = 0;
```

Comment:
I think we should add descriptions for each of these values

Reply:
Done.